app/scripts/models/notifications.js

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * The notifier broadcasts messages across multiple channels (iframe, tabs, browsers, etc).
+ */
+
+'use strict';
+
+define([
+  'backbone',
+  'underscore',
+  'lib/promise',
+  'lib/constants',
+  'lib/channels/web'
+], function (Backbone, _, p, Constants, WebChannel) {
+
+  var Notifier = Backbone.Model.extend({
+    defaults: {
```

Comment:
This is a bit strange, using the model's values to get the event strings. Would normal local constants work just as well? Another alternative is to ditch the `events` namespace and put each event as its own item. Still, it feels strange.